app/extensions/brave/content/scripts/blockFlash.js

```diff
@@ -30,7 +30,7 @@ if (adobeRegex.test(window.location.href)) {
   }
 }
 
-if (chrome.contentSettings.flashEnabled == 'allow') {
+if (chrome.contentSettings.flashEnabled == 'allow' && chrome.contentSettings.torEnabled == 'allow') {
```

Comment:
Why does this logic run only if we have flash enabled _and_ tor enabled?

Reply:
@riastradh-brave `torEnabled == 'allow'` means Tor is not enabled in the tab. 'block' means it's enabled.

my admittedly non obvious logic here was that 'allow' always corresponds to the less private/secure setting.

Reply:
Got it.  Maybe we can have a function that returns a boolean with a clearer name?